### PR TITLE
Jennifer/issue 318

### PR DESCRIPTION
**Pull request by jaube05**
About pages not scrolling fix.
Breadcrumbs styling to match uswds theme.

### CODE REVIEW

**Comment by dmundra** [comment]
Testing this locally!

File: _includes/contact.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<form action="https://usebasin.com/f/9cbe63444dee" method="POST">
+    <div class="form-input">
+        <label for="email-address">Email Address<abbr title="required" class="usa-hint usa-hint--required">*</abbr></label>
+        <input type="email" id="email" name="email-address" required>
+    </div>
+    <div class="form-input">
+        <label for="text-area">Subject<abbr title="required" class="usa-hint usa-hint--required">*</abbr></label>
+        <textarea id="body" name="text-area"></textarea>
```
Comment (dmundra) [inline _includes/contact.html:8]: @jaube05 the label `for=<value>` references the id of input and textarea and not the name. I think you just need to add the id with the same value as the name or modify it to match.

File: _includes/about.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<p>CivicActions Accessibility is an open project primarily developed by CivicActions team members.</p>
+
+<p>We follow [agile process](/agile) and encourage others to do so as well.</p>
+
+<p>We are gathering information that we find useful for our team, and hope that it resonates with our clients and the broader community. We acknolwedge that this resource is imperfect, impermanent and incomplete, and we encourage you to help make it better. There is a lot of great information about digital accessibility already freely available on the web. We aim to refer people to great resources, or build upon them.</p>
+
+<p>Please [let us know](https://accessibility.civicactions.com/contact) what you find helpful, if there are resources we should include or if there are ways we can improve.</p>
```
Comment (dmundra) [inline _includes/about.html:7]: @jaube05 because this is HTML the markdown link will need to be modified to be an HTML link on this line and line 3 above

**Review by dmundra — CHANGES_REQUESTED** [review]
@jaube05 the breadcrumbs are looking good and the changes to the about page are also working just found some markdown links on the about page and form label issues for the new contact form.

Also should I test the form input?

File: _includes/about.html [hunk 3]
```diff
@@ -1,10 +1,10 @@
 <p>CivicActions Accessibility is an open project primarily developed by CivicActions team members.</p>
 
-<p>We follow [agile process](/agile) and encourage others to do so as well.</p>
+<p>We follow <a href="https://accessibility.civicactions.com/agile">agile process</a> and encourage others to do so as well.</p>
 
 <p>We are gathering information that we find useful for our team, and hope that it resonates with our clients and the broader community. We acknolwedge that this resource is imperfect, impermanent and incomplete, and we encourage you to help make it better. There is a lot of great information about digital accessibility already freely available on the web. We aim to refer people to great resources, or build upon them.</p>
 
-<p>Please [let us know](https://accessibility.civicactions.com/contact) what you find helpful, if there are resources we should include or if there are ways we can improve.</p>
+<p>Please <a href="https://accessibility.civicactions.com/contact">let us know</a>> what you find helpful, if there are resources we should include or if there are ways we can improve.</p>
```
Comment (dmundra) [inline _includes/about.html:7]: Looks like an extra `>` was left behind. Last change and this is good to go!

**Review by dmundra — CHANGES_REQUESTED** [review]
Small typo remaining to fix.

File: _includes/about.html (hunk 3 above)
Comment (jaube05) [reply _includes/about.html:7]: Opps. Nice catch!!

**Review by dmundra — APPROVED** [review]
Thanks, @jaube05. Approved!

**Pull request merged**